localization/yabloc/yabloc_particle_filter/include/yabloc_particle_filter/prediction/experimental/suspension_adaptor.hpp

```diff
@@ -0,0 +1,100 @@
+// Copyright 2023 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef YABLOC_PARTICLE_FILTER__PREDICTION__EXPERIMENTAL__SUSPENSION_ADAPTOR_HPP_
```

Comment:
What do you mean by "suspension_adaptor"? Maybe it's better to change the name to make it more clear?

Reply:
It was a class that managed the ON/OFF state of prediction. Since it was an experimental implementation, I have removed it in  14804b8.
In the future, when implementing this feature, I will name it something like PredictionLifecycleHandler.